lib/src/main/scala/spinal/lib/tools/ModuleAnalyzer.scala

```diff
@@ -154,13 +154,16 @@ class ModuleAnalyzer(module: Module) {
     */
   def getPorts(cond: BaseType => Boolean): mutable.LinkedHashSet[BaseType] = {
     val e = new ModuleAnalyzer(module.globalData.toplevel)
-    (e.allInputs ++ e.allOutputs).filter(cond)
+    (e.getInputs ++ e.getOutputs).filter(cond)
   }
 
 }
 
 object ModuleAnalyzer {
-  implicit def toElaborator(module: Module): ModuleAnalyzer = new ModuleAnalyzer(module)
+  /** Implicitly convert Module/Component into ModuleAnalyzer */
+  implicit def toAnalyzer(module: Module): ModuleAnalyzer = new ModuleAnalyzer(module)
+
+  /** handy always-true filter */
   val cellTrue = (_: Component) => true
```

Comment:
Do you have a use-case for these two `val`?

Since the API allows to not filtering by providing no condition at all, I don't see when they should be used.